rikai/app/Http/Controllers/Admin/ChartController.php

```diff
@@ -24,4 +28,9 @@ public function index()
         return view('admin.chart.index',compact(['orders', 'data','ordersdone']));
     }
 
+    public function export() 
+    {
+        return Excel::download(new StatisticExport, 'statistic-'.date('d-m-Y').'.xlsx');
```

Comment:
dùng thêm queue nhé

Reply:
anh ơi cái này em đã thử nhiều cách với queue mà nó vẫn không được dùng hàm queue trực tiếp thì nó bảo can not found undifined funtion, còn dùng thông qua job dispatch thì nó lại không lấy được phải excel ạ

Reply:
Dữ liệu khi xuất ra nhiều không?
Nếu ít thì có thể dùng bình thường cũng được.

Reply:
dạ không anh mỗi file chỉ có 4 trường dữ liệu